Sources/OpenAPIRuntime/Conversion/ParameterStyles.swift

```diff
@@ -26,6 +26,11 @@
     ///
     /// Details: https://datatracker.ietf.org/doc/html/rfc6570#section-3.2.2
     case simple
+    
+    /// The deepObject style.
+    ///
+    /// Details: Should update datatracker
```

Comment:
Can you put a link to the RFC please?

Reply:
Unfortunately RFC doesn't refer anything regarding deepObject. My reading of OpenAPISpec[[1]](https://spec.openapis.org/oas/v3.1.0.html#style-values) is that deepObject uses form parameters configuration. 
Should we just refer to form style here too? What do you think?

## 
 1. https://spec.openapis.org/oas/v3.1.0.html#style-values

Reply:
Yeah link to the OpenAPI doc, to the closest section, that's good enoguh.